providers/implementations/macs/cmac_prov.c

```diff
@@ -87,6 +87,8 @@ static void *cmac_dup(void *vsrc)
         return NULL;
 
     dst = cmac_new(src->provctx);
+    if (dst == NULL)
+        return NULL;
```

Comment:
Side note: we really need to add more error reports...

Reply:
At what point should be start adding them?

_Now_ seems like a good answer...

Reply:
You would have to be careful.. certain tests have expectations of the error stack (evp_test for example)